### ISSUE TITLE
Saving our own rules and packing

### ISSUE THREAD

**Issue by rohithkd** [kylepaulsen/ResourceOverride#25]
I was able to create custom rules for particular websites and save the rules. I was wondering whether it is possible to save my custom rules and package it separately.

**Comment by rohithkd** [comment]
Yes. I was able to perform by this way.I could save rule and load rule through UI. But I was just wondering whether it is possible to save the rule and place it any where in source code of extension, so next time I can just load the package and my rules are intact.

**Comment by kylepaulsen** [comment]
Oh, in that case, no, not currently. The solution would probably use account storage to store your rules.

**Comment by rohithkd** [comment]
So currently where the rules are getting stored? It is observed that the rules are saved even on reboot. This striked me the possibility of saving the rule and using it next time on loading the extension.

**Comment by kylepaulsen** [comment]
Wait are you saying that your rules are not being persisted though a reboot? They definitely should be. Yes the rules are stored in the extension's indexedDB (on disk). If they are not being saved through a reboot, I wonder if any of your chrome profile data is. Or it could be a bug but that would be a huge one that no one has seen yet.

**Comment by rohithkd** [comment]
Really sorry that my wordings confused you. 
The rules are persisted even on reboot.

What I meant was that instead of storing on extension indexedDB, I thought there is any option to save in the extension folder. So that I can use this in another system without manually loading the rule, and just unloading the extension would do for me.

From the conversation, I now understand that it is not possible and would probably need account storage. 

We can tag this issue as "feature request" and see that it is possible to implement this idea.

**Comment by chrishumphries** [comment]
I am also interested in this feature - any updates here?
